docs/releases/3.0.txt

```diff
@@ -153,6 +153,10 @@ Management Commands
 * The new :option:`compilemessages --ignore` option allows ignoring specific
   directories when searching for ``.po`` files to compile.
 
+* The :option:`startapp --template` and :option:`startproject --template`
+  options now support templates stored in lzma-compressed tarfiles
+  (`*.tar.xz`).
```

Comment:
Use double backticks and update the list of extensions in `docs/ref/django-admin.txt`.

Reply:
Ah, yes.